dist/wsdl-lib/IscrizioniAvvisaturaService/PPTPort.d.ts

```diff
@@ -23,9 +23,9 @@ export interface IPPTPortSoap {
 export declare namespace PPTPortTypes {
     interface IidentificativoUnivocoSoggetto {
         /** http://ws.pagamenti.telematici.gov/#stTipoIdentificativoUnivocoPersFG(length,F,G) */
-        tipoIdentificativoUnivoco: "F" | "G";
+        "sac:tipoIdentificativoUnivoco": "F" | "G";
```

Comment:
is this code generated automatically ? (then patched)
will be there any issue re-generating it in case ? (how to keep this patch in sync)
eventually, we should patch this with a post-generation script

Reply:
It is generated automatically. We're looking for an automatic process to fix this issue or some modifications to the existing script/wsdl. For now, this pull request was made to realize a "custom version" of italia-pagopa-api to integrate the module in "italia-pagopa-proxy"